rovercode/drivers/grovepi_chainable_rgb_leds.py

```diff
@@ -0,0 +1,50 @@
+"""Class for communicating with a GrovePi chainable RGB LED."""
+
+import os
+import logging
+import time
+
+logging.basicConfig()
+LOGGER = logging.getLogger(__name__)
+LOGGER.setLevel(logging.getLevelName('INFO'))
+if os.getenv('DEVELOPMENT', 'false').lower() == 'true':
+    LOGGER.warning("In DEVELOPMENT mode. Using dummy.")
+    from drivers.dummy_grovepi_interface import chainableRgbLed_pattern, \
+        storeColor, chainableRgbLed_init, pinMode
+else:
+    from grovepi import chainableRgbLed_pattern, \
+        storeColor, chainableRgbLed_init, pinMode
+
+
+class GrovePiChainableRgbLeds:
+    """A module to set the GrovePi Chainable RGB LEDs."""
+
+    MODE_THIS_LED_ONLY = 0
+    COMPONENT_RANGE = 0, 256
```

Comment:
It seems like this would need to be `range(0, 256)`. Otherwise, the checks are just looking for it to be `0` or `256`

Reply:
Ah, I fixed this in the test mock but forgot to do it here!